From 1fe7ec75a9d850deb70f73f64514876010b38dd9 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Wed, 10 Dec 2014 19:57:14 +0800
Subject: [PATCH] ARM64: rockchip: update rockchip_defconfig

-MEMCG
-RD_BZIP2
-RD_LZMA
PANIC_TIMEOUT=1
+BLOCK_RKNAND
+ARMV7_COMPAT
+SECCOMP
-MALI_MIDGARD
+LCDC_RK3368
+SND_DYNAMIC_MINORS
+BOOTPARAM_HARDLOCKUP_PANIC
+BOOTPARAM_SOFTLOCKUP_PANIC
DEFAULT_HUNG_TASK_TIMEOUT=10
+DYNAMIC_DEBUG

Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 23 ++++++++++-------------
 1 file changed, 10 insertions(+), 13 deletions(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index c544e1513df6..b66d5aac046a 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -10,8 +10,6 @@ CONFIG_CGROUP_DEBUG=y
 CONFIG_CGROUP_FREEZER=y
 CONFIG_CGROUP_CPUACCT=y
 CONFIG_RESOURCE_COUNTERS=y
-CONFIG_MEMCG=y
-CONFIG_MEMCG_SWAP=y
 CONFIG_CGROUP_SCHED=y
 CONFIG_RT_GROUP_SCHED=y
 CONFIG_NAMESPACES=y
@@ -20,10 +18,8 @@ CONFIG_NAMESPACES=y
 # CONFIG_PID_NS is not set
 CONFIG_RELAY=y
 CONFIG_BLK_DEV_INITRD=y
-CONFIG_RD_BZIP2=y
-CONFIG_RD_LZMA=y
 CONFIG_CC_OPTIMIZE_FOR_SIZE=y
-CONFIG_PANIC_TIMEOUT=5
+CONFIG_PANIC_TIMEOUT=1
 CONFIG_KALLSYMS_ALL=y
 CONFIG_EMBEDDED=y
 CONFIG_PERF_EVENTS=y
@@ -35,13 +31,16 @@ CONFIG_MODULE_UNLOAD=y
 CONFIG_PARTITION_ADVANCED=y
 CONFIG_CMDLINE_PARTITION=y
 CONFIG_ARCH_ROCKCHIP=y
+CONFIG_BLOCK_RKNAND=y
 CONFIG_SMP=y
 CONFIG_SCHED_MC=y
 CONFIG_PREEMPT=y
+CONFIG_ARMV7_COMPAT=y
 CONFIG_KSM=y
 CONFIG_DEFAULT_MMAP_MIN_ADDR=32768
 # CONFIG_CROSS_MEMORY_ATTACH is not set
 CONFIG_CMA=y
+CONFIG_SECCOMP=y
 # CONFIG_COREDUMP is not set
 CONFIG_COMPAT=y
 CONFIG_PM_AUTOSLEEP=y
@@ -52,7 +51,6 @@ CONFIG_PM_RUNTIME=y
 CONFIG_PM_DEBUG=y
 CONFIG_WQ_POWER_EFFICIENT_DEFAULT=y
 CONFIG_SUSPEND_TIME=y
-CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_DEFAULT_GOV_INTERACTIVE=y
 CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
 CONFIG_CPU_FREQ_GOV_POWERSAVE=y
@@ -368,13 +366,6 @@ CONFIG_MEDIA_RC_SUPPORT=y
 CONFIG_MEDIA_CONTROLLER=y
 CONFIG_VIDEO_V4L2_SUBDEV_API=y
 CONFIG_V4L_PLATFORM_DRIVERS=y
-CONFIG_MALI_MIDGARD=m
-CONFIG_MALI_MIDGARD_DVFS=y
-CONFIG_MALI_MIDGARD_RT_PM=y
-CONFIG_MALI_MIDGARD_DEBUG_SYS=y
-CONFIG_MALI_EXPERT=y
-CONFIG_MALI_DEBUG_SHADER_SPLIT_FS=y
-CONFIG_MALI_PLATFORM_THIRDPARTY_NAME="rk"
 CONFIG_FB=y
 CONFIG_BACKLIGHT_LCD_SUPPORT=y
 # CONFIG_LCD_CLASS_DEVICE is not set
@@ -382,6 +373,7 @@ CONFIG_BACKLIGHT_CLASS_DEVICE=y
 # CONFIG_BACKLIGHT_GENERIC is not set
 CONFIG_BACKLIGHT_PWM=y
 CONFIG_FB_ROCKCHIP=y
+CONFIG_LCDC_RK3368=y
 CONFIG_RK_TRSM=y
 CONFIG_DP_ANX6345=y
 CONFIG_RK_HDMI=y
@@ -392,6 +384,7 @@ CONFIG_LOGO=y
 # CONFIG_LOGO_LINUX_VGA16 is not set
 CONFIG_SOUND=y
 CONFIG_SND=y
+CONFIG_SND_DYNAMIC_MINORS=y
 # CONFIG_SND_SUPPORT_OLD_API is not set
 CONFIG_SND_VERBOSE_PRINTK=y
 # CONFIG_SND_DRIVERS is not set
@@ -575,10 +568,14 @@ CONFIG_PRINTK_TIME=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_DEBUG_SECTION_MISMATCH=y
 CONFIG_LOCKUP_DETECTOR=y
+CONFIG_BOOTPARAM_HARDLOCKUP_PANIC=y
+CONFIG_BOOTPARAM_SOFTLOCKUP_PANIC=y
+CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=10
 CONFIG_SCHEDSTATS=y
 CONFIG_TIMER_STATS=y
 # CONFIG_DEBUG_PREEMPT is not set
 CONFIG_SCHED_TRACER=y
+CONFIG_DYNAMIC_DEBUG=y
 CONFIG_KEYS=y
 CONFIG_SECURITY=y
 CONFIG_SECURITY_NETWORK=y
-- 
2.35.3

